mithril-aggregator/src/commands/genesis_command.rs

```diff
@@ -45,6 +45,9 @@ pub enum GenesisSubCommand {
 
     /// Genesis certificate bootstrap command.
     Bootstrap(BootstrapGenesisSubCommand),
+
+    /// Genesis keypair generation command.
+    GenerateKeypair(GenerateKeypairGenesisSubCommand),
```

Comment:
Would adding an alias be interesting?

```suggestion
    /// Genesis keypair generation command.
    #[clap(alias("gkp"))]
    GenerateKeypair(GenerateKeypairGenesisSubCommand),
```

Reply:
It is not necessary IMO as this command will likely be used rarely.